pkgs/development/libraries/fltk/1.4.nix

```diff
@@ -25,19 +25,26 @@
   withCairo ? true,
   cairo,
 
-  withPango ? stdenv.hostPlatform.isLinux,
+  withPango ? withXorg,
```

Comment:
If we disable Xorg and want to use Wayland instead, should this be disabled? Does pango not work on Wayland?

Reply:
Added an assert to make this clearer